### PR TITLE
Follow-up to #831.

### PR DESCRIPTION
**Pull request by novemberborn**
Follow-up to #831.

### CODE REVIEW

File: lib/test.js [hunk 1]
```diff
@@ -191,7 +191,7 @@ Test.prototype._result = function () {
 	if (this.metadata.failing) {
 		passed = !passed;
 		if (!passed) {
-			reason = new Error('Test was expected to fail, but succeeded, you should unmark the test as failing');
+			reason = new Error('Test was expected to fail, but succeeded. You should stop marking the test as failing');
```
Comment (sindresorhus) [inline lib/test.js]: It's kinda weird only having a `.` after the first sentence.

Comment (sindresorhus) [inline lib/test.js]: We might want an ESLint rule for this.

Comment (novemberborn) [inline lib/test.js]: OK, reverted to the original `succeeded, you should` form.

ESLint rule?

Comment (sindresorhus) [inline lib/test.js]: https://github.com/sindresorhus/eslint-plugin-xo/issues/26

**Pull request merged**